src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/DefaultRazorFormattingServiceTest.cs

```diff
@@ -0,0 +1,383 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
+{
+    public class DefaultRazorFormattingServiceTest : FormattingTestBase
+    {
+        [Fact]
+        public async Task FormatsCodeBlockDirective()
```

Comment:
Should probably sprinkle a few `@code`'s instead of `@functions` in some of these existing tests